### ISSUE TITLE
[Bug]: Visual Incompatibility with Polytone Fog resource pack(s?)

### ISSUE THREAD

**Issue by ErakAcheron** [nonamecrackers2/simple-clouds#81]
### Guidelines

- [X] By checking this box, I am saying that I have read the guidelines and believe to be following them to the best of my ability

### Minecraft Version

1.21.1

### Mod Version

0.6.3

### Mod Loader

NeoForge

### Mod Loader Version

21.1.77

### Graphics Card

NVIDIA GeForce RTX 3060

### Describe the Bug

Simple clouds causes rendering issues with a Polytone resource pack that modifies fog, screenshots attached:
With SimpleClouds and the Polytone pack active:
![2024-11-14_09 40 28](https://github.com/user-attachments/assets/a4197a62-a0d8-488e-9577-a08d79c5d205)
The Polytone pack without SimpleClouds:
![2024-11-14_09 42 57](https://github.com/user-attachments/assets/b8d557bf-948d-47ad-9bcb-93be9a54f613)
SimpleClouds without the Polytone pack active:
![image](https://github.com/user-attachments/assets/9c05b926-f384-4e15-a861-b2e17034e185)

Link to resource pack in question: https://modrinth.com/resourcepack/fogulous

### Steps to Reproduce

1. Install SimpleClouds and Polytone on NeoForge 1.21.1
2. Load the resource pack Fogulous 
3. Observe visual conflicts with SimpleClouds fog

### Crash Reports/Logs

[debug.log](https://github.com/user-attachments/files/17739431/debug.log)

### Screenhots

One issue that really stands out is the fog conflict on water within a storm:
![2024-11-14_10 20 15](https://github.com/user-attachments/assets/7c17a737-52d8-454b-9602-dcae444ca060)
Another is storm fog overlaying terrain:
![2024-11-14_10 22 52](https://github.com/user-attachments/assets/65d19faf-7e9e-4c89-b8b0-047c7ecb5119)

### Mod List

Description screenshots were taken with sodium installed, but removing **all** other mods showed no discernible difference.

**Comment by ErakAcheron** [comment]
I should mention, this is with screen_space fog mode.

**Comment by nonamecrackers2** [comment]
Yeah this is because of that fog mode. Not sure how else to really fix this for now, but you can change the fog mode to prevent this from happening.

**Comment by VirtualGaming1** [comment]
@ErakAcheron one thing first did you make sure you have Polytone mod as well that is required to have as well if so I think this is more of compatibility side of Polytone as I kinda think it is like a shader type mod (I am not sure but feel like it is what shaders do as well.)

NOTE: I ask if you have Polytone enabled is because I do not see it on your mod list

**Comment by Nyoom-Monster** [comment]
NOTE: This is the second time I have had Simple Clouds installed in this pack. The first time, my game FPS was so low I decided to remove it, and I had not installed Polytone yet, but did have Atmospheric Fog (Fogulous). Even with clouds turned off in vanilla settings, my FPS was bad.

I'm having trouble with all three of these in a pack together, too. I'm running a modified version of a modpack called Create: Forge Frontier on Forge in 1.20.1 (loaded by Curseforge), and when I loaded my world after having configured everything for Simple Clouds beforehand, what I'm seeing is basically just the blue vanilla skybox, my arm, no HUD, and error messages in the chat saying the game couldn't retrieve the questbook because FTB Quests and FTB Teams couldn't be found. My game remained frozen, while the vanilla output log, which I have Curseforge load every time I start an instance, initially got stuck, but after a few minutes went back to the normal loading procedures, but was marking down small warnings/errors with object models, and reporting that processes were ticking for too long.

I ended up having to stop the instance with Task Manager, because it wouldn't crash itself. I thought maybe the compat mod I had installed was the issue, because I don't think I had installed it before when I originally added Simple Clouds to the pack the first time, but I also installed Polytone this time around, which I had only just realized I hadn't done yet for Atmospheric Fog (Fogulous), so unless getting rid of the compat mod fixes this, I think I'm having the same issue.

Let me know if I should fill out my own report.

*The label 1.20.1 is wrong, it should be 1.21.1 according to the person who posted this.

(Thank you for posting this, by the way. I have the port of Fogulous, Atmospheric Skies, and Polytone in my pack with Simple Clouds, and without this post I would have no clue what was going on. The problem I'm having is different, but I originally thought it was due to installing the Iris/Oculus compat with it.)
